### PR TITLE
Use ansible to configure vault

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: ruromero <rromerom@redhat.com>

Deploy Vault using kustomize on ArgoCD, then initialize it using Ansible.
I have also fixed the post-deploy job script to avoid printing the password in the logs.

### CODE REVIEW

**Comment by ruromero** [comment]
@mpaulgreen ready to be reviewed

File: README.md [hunk 1]
```diff
@@ -76,10 +78,156 @@ to update the repository path or branch where the charts will be installed from.
...
+token_policies                      [argocd-repo-server default]
+token_ttl                           15m
+token_type                          default
+ttl                                 15m
+```
+
+* Validate the ArgoCD - Vault integration works
+
+```bash
+$ REPO_SVR_POD=$(oc get po -l app.kubernetes.io/name=redhat-kong-gitops-repo-server -ojson | jq -r '.items[0].metadata.name')
```
Comment (mpaulgreen) [inline README.md:215]: ```oc -n openshift-gitops get po -l app.kubernetes.io/name=redhat-kong-gitops-repo-server -ojson | jq -r '.items[0].metadata.name'```

File: README.md [hunk 2]
```diff
@@ -76,10 +78,156 @@ to update the repository path or branch where the charts will be installed from.
...
+$ oc cp gateway/prereqs/base/vault/license-sercret.yaml ${REPO_SVR_POD}:secret.yaml
```
Comment (mpaulgreen) [inline README.md:216]: ```oc -n openshift-gitops cp gateway/prereqs/base/vault/license-sercret.yaml ${REPO_SVR_POD}:secret.yaml```

**Review by mpaulgreen — APPROVED** [review]
Approved

**Pull request merged**